deviceplugin/generic.go

```diff
@@ -91,6 +91,7 @@ type device struct {
 // * be found using either a file path or a USB identifier; and
 // * mounted and used without special logic.
 type GenericPlugin struct {
+	v1beta1.UnimplementedDevicePluginServer
```

Comment:
What's the reason for embedding this?

Reply:
There was a change in more recent versions of grpc-go that basically requires this, see also:
* https://github.com/grpc/grpc-go/pull/3657
* https://github.com/grpc/grpc-go/issues/3669